testproject/Assets/Tests/Manual/NetworkAnimatorTests/AnimatedCubeController.cs

```diff
@@ -229,6 +243,10 @@ private void LateUpdate()
                 BeginAttack(2);
             }
 
+            if (Input.GetKeyDown(KeyCode.Alpha3))
```

Comment:
This too

Reply:
It is part of the manual test that just allows you to verify that when changes are applied to a layer's weight that it was updated on other non-authoritative/owner instances.